.github/workflows/ci.yml

```diff
@@ -24,6 +27,24 @@ jobs:
           php-version: '8.1'
           tools: composer:v2
 
+      - name: Get Composer cache directory
+        id: composer-cache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
```

Comment:
`set-output` is deprecated and GitHub recommends to use slightly different way to write ti output - https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Reply:
@Nikschavan Thank you for catching that one up. I've addressed it in 530a355.